docs/Dependency.md

```diff
@@ -0,0 +1,93 @@
+# Microsoft Application Insights JavaScript SDK - Dependencies Plugin
+
```

Comment:
This file moves dependency section to docs, will update main readme when this docs is merged